### ISSUE TITLE
Migration commands need to be run before starting image, api and auth containers

### ISSUE THREAD

**Issue by ToshKoevoets** [Amsterdam/openstad-kubernetes#29]
I've tried adding migrate command in deployment files, but doesn't seem to work.

In k8s/deployment/api.yaml

```
command: ['node migrate.js', 'npm start']
```

For dev environment I've solved it for now by adding command to npm start. Not sure what is better.

```
node migrate.js
```

For api it is done
```
node migrate.js
```

For image and auth it is managed with
```
knex migrate:latest
```

**Comment by diederikgithub** [comment]
@ToshKoevoets : The migrations are already done during container initialization with the initContainers command. So I think there should be no need for this?

**Comment by ToshKoevoets** [comment]
Migrations need to be run every update of container. This doesnt seem to
happen now. Api errors unless I add Migration script. Init-db is only on
creating right?

**Comment by diederikgithub** [comment]
Updating the image must trigger the initContainers, so I'm not sure what is wrong. Tried it myself with:
`kubectl set image deployment/openstad-api openstad-api=openstad/api:latest --record` 
and the logs say:
`+ openstad-api-7447dd6c77-rzk5m › init-db-ready`
`openstad-api-7447dd6c77-rzk5m init-db-ready No new migrations`

The change course is recorded (kubectl get deployment openstad-api -o yaml):
` kubernetes.io/change-cause: kubectl set image deployment/openstad-api openstad-api=openstad/api:latest
      --record=true
 `
Could you check that with your deployment?

**Comment by ToshKoevoets** [comment]
Okay will have to debug then, deploy with travis only started working when I add node migrate.js to the npm start command.

**Comment by diederikgithub** [comment]
@ToshKoevoets Did you manage to debug it?
